.github/workflows/pr-build.yaml

```diff
@@ -17,6 +17,7 @@ jobs:
     env:
       # Set LANG=C to simulate least-common-denominator target deployment environments:
       LANG: C
+      SCALANATIVE_THREAD_STACK_SIZE: 10m
```

Comment:
note: to prevent stackoverflow in native tests https://github.com/scala-native/scala-native/issues/4253